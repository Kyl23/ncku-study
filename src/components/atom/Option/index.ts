import styled from '@emotion/styled';
import MenuItem from '@mui/material/MenuItem';

export default styled(MenuItem)`
    font-size: 1.8rem;
`;
