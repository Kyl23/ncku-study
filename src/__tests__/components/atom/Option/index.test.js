import { render } from '@testing-library/react';
import Option from '~/src/components/atom/Option';

describe('test for atom - Option', () => {
    it('render Option must always be same', () => {
        const { container } = render(<Option />);

        expect(container).toMatchSnapshot();
    });
});
