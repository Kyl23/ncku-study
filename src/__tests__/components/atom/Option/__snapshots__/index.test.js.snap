// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for atom - Option render Option must always be same 1`] = `
<div>
  <li
    class="MuiMenuItem-root MuiMenuItem-gutters MuiButtonBase-root css-19wq24w-MuiButtonBase-root-MuiMenuItem-root"
    role="menuitem"
    tabindex="-1"
  >
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </li>
</div>
`;
